src/frontend/components/UI/Dialog/components/Dialog.tsx

```diff
@@ -28,7 +28,7 @@ const StyledPaper = styled(Paper)(() => ({
   color: 'var(--text-default)',
   maxWidth: '100%',
   '&:has(.settingsDialogContent)': {
-    height: '80%'
+    maxHeight: '80%'
```

Comment:
a fixed height made the logs dialog too tall

![image](https://github.com/user-attachments/assets/991cf086-8c63-4317-98d9-e29730c0cb3e)

and now:
![image](https://github.com/user-attachments/assets/26e971c6-bcfb-4d1e-8906-37a03380ff98)